zict/common.py

```diff
@@ -23,16 +21,16 @@ def update(*args, **kwds):
         if args:
             other = args[0]
             if isinstance(other, Mapping) or hasattr(other, "items"):
-                items += other.items()
+                items = other.items()
```

Comment:
It looks like now `items` are a `dict_items` type, but we are still initializing it as a list on line 20. Is this ok, does it matter?

Reply:
The list is just a dummy empty iterable to simplify chaining on line 48. It's never filled.